solidity/contracts/LPRewards.sol

```diff
@@ -214,3 +221,27 @@ contract LPRewards is LPTokenWrapper, IRewardDistributionRecipient {
         }
     }
 }
+
+/// @title KEEP rewards for TBTC-ETH liquidity pool.
+contract LPRewardsTBTCETH is LPRewards {
+    constructor(KeepToken keepToken, IERC20 tbtcEthUniswapPair)
```

Comment:
The first parameter could be also `IERC20`. I'm not sure what's better, use specific `KeepToken` type or more general `IERC20`?

```suggestion
    constructor(IERC20 keepToken, IERC20 tbtcEthUniswapPair)
```

Reply:
Given that those contracts are for specific pairs, I think it makes more sense to keep the code as-is.